Formula/c/cyctl.rb

```diff
@@ -0,0 +1,43 @@
+class Cyctl < Formula
+  desc "Customizable UI for Kubernetes workloads"
+  homepage "https://cyclops-ui.com/"
+  url "https://github.com/cyclops-ui/cyclops/archive/refs/tags/v0.6.2.tar.gz"
+  sha256 "02ed62bec983a9937ec6decb4185c405c5a2da41adaa056b30c846d9161cf46e"
+  license "Apache-2.0"
+  head "https://github.com/cyclops-ui/cyclops.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    cd "cyctl" do
+      system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/cyclops-ui/cycops-cyctl/common.CliVersion=#{version}")
+    end
+  end
```

Comment:
can we add UI as well? (there are some examples in the codebase that also build UI)

Reply:
> can we add UI as well? (there are some examples in the codebase that also build UI)

This time, we plan to submit the `cyctl` for module management.

The UI component may be included in a future PR.